src/styles.scss

```diff
@@ -1 +1,61 @@
 /* You can add global styles to this file, and also import other style files */
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@600%3B700%3B800&display=swap");
+@import "@angular/material/prebuilt-themes/indigo-pink.css";
+@import "~@ng-select/ng-select/themes/default.theme.css";
+
+$content-width: 1020px;
```

Comment:
Are you hardcoding width device? What if it will be smaller or bigger?

Reply:
I have added the variable to stay accurate with the design from figma, but now I have changed the approach from directly setting element's width to this value to set a max-width: $content-width; and width:100%; That way when the page is smaller the content shrinks more naturally.